### PR TITLE
Fix for deleting database with '/' in name

### PR DESCRIPTION
**Pull request by benkeen**
Previously you'd get an error when trying to delete a database
that contained a slash in the name.

### CODE REVIEW

File: app/addons/documents/views.js [hunk 1]
```diff
@@ -196,7 +196,7 @@ function (app, FauxtonAPI, Components, Documents,
         clear: true
       });
 
-      this.database.url = FauxtonAPI.urls('databaseBaseURL', 'server', this.database.id, '');
+      this.database.url = FauxtonAPI.urls('databaseBaseURL', 'server', encodeURIComponent(this.database.id), '');
```
Comment (robertkowalski) [inline app/addons/documents/views.js]:
```js
this.database.saveID()
```

what do you think?

Comment (robertkowalski) [inline app/addons/documents/views.js]: https://github.com/apache/couchdb-fauxton/blob/b3e5b44e278c6aa750eb21c1b2f1025ede83c9a8/app/addons/databases/resources.js#L77

Comment (benkeen) [inline app/addons/documents/views.js]: Yeah that's better. Will update in a sec.

**Comment by michellephung** [comment]
+1

**Comment by benkeen** [comment]
Merged. (I squashed, hence the re-running of the test - but it was green)

**Pull request merged**